doc/bip135-guide.md

```diff
@@ -0,0 +1,96 @@
+# BIP135 Guide
+
+There is no consensus for the November hardfork. Both BitcoinABC (ABC) and BitcoinSV (SV) plan to activate a set of specific rule changes on November 15th.  Although Bitcoin Unlimited has been involved in the development of some of these features, the impetus to force contentious hard forks is coming from the ABC and SV groups, so in this document we will call them the _ABC feature set_ and the _SV feature set_ for simplicity, without implying sole authorship.
+
+Bitcoin Unlimited (BU) provides an option to compromise. The interim release (0.15.0.0) enables BIP135 voting for each of the proposed changes, and contains the ABC feature set.  The next release will soon enable further configuration options and deliver the SV feature set.
+
+BIP135 voting is only meant for miners, since it only affects the **_version_** field in mined blocks.
+
+
+## BIP135 VOTING
+
+With BIP135 a miner can vote for individual features of the ABC and SV November forks. This is meant as an option to let miners compromise on the incompatible forks, for example by activating uncontroversial features but delaying controversial changes.
+
+It is important to understand that a BIP135 vote for a feature is a vote of **_NO_** for the November fork, but a vote of **_Yes_** for that feature to be activated via BIP135 at some later time.  
+
+If you do not support contentious hard forks, please start using BIP135 voting!  Even if you are mining with a different client, your mining pool can do so by changing the block‘s **_version_** field.
```

Comment:
bold this first sentence ("if you do not support ... voting!").  If the reader can only take away 1 thing, this is what we want him/her to learn.